code/game/objects/items/cards_ids.dm

```diff
@@ -648,6 +636,12 @@ update_label("John Doe", "Clowny")
 			return
 	return ..()
 
+/obj/item/card/id/syndicate/examine(mob/user)
+	if(!user?.mind?.special_role)
+		// Non-syndicate see a generic/fake description
+		to_chat(user, span_notice("This ID card belongs to [registered_name || "John Doe"], [assignment || "Master at Arms"]."))
+		return list()
```

Comment:
Return the description as part of the list, rather than as a to_chat so that it is formatted the same way as normal examines. This also fails to handle other things on the ID such as bank account, and doing it with the previous method of `update_label` may be more effective.

Reply:
I dont think the examine override is necessary tbh